common/src/main/java/com/wynntils/screens/base/widgets/ImportButton.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.base.widgets;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.screens.base.TooltipProvider;
+import com.wynntils.utils.render.RenderUtils;
+import com.wynntils.utils.render.Texture;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+
+public class ImportButton extends WynntilsButton implements TooltipProvider {
```

Comment:
> implements TooltipProvider
haven't seen this one before, I usually manually rendered them.. Do they work as they are supposed to?